js/coinspot.js

```diff
@@ -278,6 +279,50 @@ module.exports = class coinspot extends Exchange {
         return this.parseTicker (ticker, market);
     }
 
+    async fetchTickers (symbols = undefined, params = {}) {
+        /**
+         * @method
+         * @name coinspot#fetchTickers
+         * @description fetches price tickers for multiple markets, statistical calculations with the information calculated over the past 24 hours each market
+         * @see https://www.coinspot.com.au/api#latestprices
+         * @param {[string]|undefined} symbols unified symbols of the markets to fetch the ticker for, all market tickers are returned if not assigned
+         * @param {object} params extra parameters specific to the coinspot api endpoint
+         * @returns {object} an array of [ticker structures]{@link https://docs.ccxt.com/en/latest/manual.html#ticker-structure}
+         */
+        await this.loadMarkets ();
+        const response = await this.publicGetLatest (params);
+        //
+        //    {
+        //        "status": "ok",
+        //        "prices": {
+        //        "btc": {
+        //        "bid": "25050",
+        //        "ask": "25370",
+        //        "last": "25234"
+        //        },
+        //        "ltc": {
+        //        "bid": "79.39192993",
+        //        "ask": "87.98",
+        //        "last": "87.95"
+        //        }
+        //      }
+        //    }
+        //
+        const result = {};
+        const prices = this.safeValue (response, 'prices');
+        const ids = Object.keys (prices);
+        const timestamp = this.milliseconds ();
```

Comment:
Hummm @AlgoQ I don't like this, `this.milliseconds ();` 
request time might be very different from the "real" update time, so I think we should just leave it as undefined